protocol/git/README.md

```diff
@@ -47,8 +47,9 @@ Write a [good commit message]. Example format:
 
     http://project.management-system.com/ticket/123
 
-If you've created more than one commit, use a rebase to squash them into
-cohesive commits with good messages:
+If you've created more than one commit, use a
+[rebase to squash](https://davidwalsh.name/squash-commits-git)
```

Comment:
Thoughts on linking to git documentation directly instead? https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages